netket/graph/graph.py

```diff
@@ -163,8 +160,22 @@ def Edgeless(nodes):
             4
     """
     if not isinstance(nodes, list):
-        raise TypeError("nodes must be a list")
+        if not isinstance(nodes, int):
+            raise TypeError("nodes must be either an integer or a list")
+        nodes = range(nodes)
 
     edgelessgraph = _nx.MultiGraph()
     edgelessgraph.add_nodes_from(nodes)
     return NetworkX(edgelessgraph)
+
+
+def DoubledGraph(graph):
```

Comment:
more a reminder for me than anything: this doubled graph does not correctly set automorphisms...